src/css/components/_poster.scss

```diff
@@ -30,3 +30,8 @@
 .vjs-using-native-controls .vjs-poster {
   display: none;
 }
+
+// Don't hide the poster when audio-poster-mode is true
+.vjs-has-started.vjs-audio-poster-mode .vjs-poster {
+  display: block;
+}
```

Comment:
thought: Could this be combined with the above `.vjs-audio.vjs-has-started .vjs-poster` block? Seems like it'd be cleaner that way. Could probably also combine the `.vjs-has-started .vjs-poster` and `.vjs-using-native-controls .vjs-poster` blocks, too.